# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../glibc/mips64.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We do not use CFLAGS to pass options around, we have wrappers for the
reliability of it. So build for 64bit when we ask for mips64-...

  - Rene Rebe <rene@exactcode.de>

--- glibc-2.11/ports/sysdeps/mips/preconfigure.orig	2010-09-23 20:48:14.198691801 +0000
+++ glibc-2.11/ports/sysdeps/mips/preconfigure	2010-09-23 20:49:36.743635974 +0000
@@ -4,7 +4,7 @@
 		*" -mabi=n32 "*) mips_cc_abi=n32 ;;
 		*" -mabi=64 "*|*" -mabi=n64 "*) mips_cc_abi=64 ;;
 		*" -mabi=32 "*|*" -mabi=o32 "*) mips_cc_abi=32 ;;
-		*) mips_cc_abi=default ;;
+		*) mips_cc_abi=64 ;;
 		esac
 		case $config_os in
 		*abin32*) mips_config_abi=n32 ;;
